charts/fluent-operator/values.yaml

```diff
@@ -22,9 +24,10 @@ operator:
       requests:
         cpu: 50m
         memory: 64Mi
-  container:
-    repository: "kubesphere/fluent-operator"
-    tag: "latest"
+  image:
+    registry: ghcr.io
```

Comment:
The registry field allows users to use a different registry e.g. ecr-pullthrough or mirroring the images in one of their own registries.